kf_lib_data_ingest/etl/ingest_pipeline.py

```diff
@@ -76,6 +77,7 @@ def __init__(
         self.ingest_output_dir = os.path.join(self.ingest_config_dir, 'output')
 
         self.target_api_config_path = target_api_config_path
+        self.auth_configs = auth_configs
```

Comment:
Make sure to pop auth_configs from kwargs before logging ingest parameters. Could potentially have secrets and we don't want to log secrets